### DISCUSSION TITLE
Validation error for Atom, OpenSearch

### DISCUSSION THREAD

**Discussion by tfunq** [INSPIRE-MIF/helpdesk-validator#838] [General]
Dear,
Testing our ATOM feed, https://opendata-download-icemap-utv.smhi.se/inspire.xml, gives error on in "OpenSearch Description", "Iterate over Get Spatial Dataset URL". See https://inspire.ec.europa.eu/validator/test-run/details.html?id=EIDad2171e2-782d-469f-aa06-b5cc83ad1670. The error message is `java.lang.NullPointerException`, `error at line: 75`. Can anyone clarify why I get that error? And is it possiblie to get more logging when executing tests? In the "Test instruction" available there are a lot of info-logging that's not available in the log file.
Regards,
Tomas

**Comment by HerzovanderWal** [comment]
Yes, It checks if it is http or https in the URL. It expects http://

**Comment by fabiovinci** [comment]
Dear @tfunq,

your ATOM feed is no longer available, do you still have the error?

**Reply by tfunq** [reply]
Sorry. The URL pointed towards our development-environment. It should be https://opendata-download-icemap.smhi.se/inspire.xml. But the problem does not exist anymore.
